src/pages/blog/how-to-thrive-in-an-automated-society.pug

```diff
@@ -0,0 +1,184 @@
+
+            p By investing in your own bespoke automation software to connect these systems, you can fully automate much of your businesses operation, and create unique intellctual property in the process. A common example of this is integrating the data from your CRM product, your web portal, and your analytics product together.
+
+            p Integrating off the shelf automation products with one another improves scalability, improves data quality, reduces data duplication, and even makes your business easily programmable for new internal products, or other B2B arrangements via an API.
+
+            h4 Niche and scale
+            p Traditional logic tells us that in order to thrive, should remain generalists and attract as much business as possible. Unfortunately, it's hard to grow a business that caters to everyone's needs an automated way, and if you do succeed, you end up blending into a crowd of countless competitors that look the same.
+
+            p Picking a niche makes it easier to automate, as your customers increasingly fit a similar mold that you can build robust processes around. You can then scale this while standing out from the crowd as the clear choice for your particular niche.
+
+            p At a global scale, you can build a decent sized business around even relatively obscure niches, and if you want to grow even further, you can begin to cater for a broader range of customers from a position of notoriety and strength.
+
+            h4 Embrace new revenue streams
+            p As you automate, you'll free up time for your business to work on other things. You can compound the benefit you get from automation by re-using existing automated processes to cater for new markets, or building new products entirely. Common examples include:
+
+            ul
+              li Charging other organisations to integrate with your services on a self-service basis.
+              li Becoming established in a niche at a global level, and <em>then</em> beginning to generalise.
+              li Building entirely new products that complement your existing product offerings.
+              li Teaching, and consulting for other businesses.
+              li Selling customer information in an ethical way.
+
+            p This not only allows you to grow your business, but helps to diversify your revenue to protect against uncertainty in the future.
+
+            h4 Focus on the 80%
+
+            p The Pareto Principle states that roughly 80% of effects come from 20% of causes. While it's not a hard rule, it applies in an uncanny way to all kinds of situations. For example, in automated businesses, it's normal for 80% of the manual work to come from 20% (or fewer) of the customers.
```

Comment:
**Pareto Principle:** Should this link to (in a new tab) a Google search or an article about the Pareto Principle?